api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/audit/EventAuditor.java

```diff
@@ -40,6 +41,19 @@ public void auditNewClaim(Claim claim) {
         eventLogger.logEvent(newClaimEvent);
     }
 
+    /**
+     * Audit an updated claim.
```

Comment:
How about "Audit that a claim has been updated"?

Reply:
Done.